components/scheduler/views/agenda.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Agenda
+page_title: Scheduler - Agenda View
+description: Agenda View in the Scheduler for Blazor.
+slug: scheduler-views-agenda
+tags: telerik,blazor,scheduler,view,agenda
+published: True
+position: 6
+---
+
+# Agenda View
+
+The Agenda view of the Scheduler for Blazor shows a weekly summary (or a custom period set by the user) in a table format. Also the Agenda view can be configured to display the events that are [grouped by resource]({%slug scheduler-resource-grouping%})
+
+
+In this article:
+
+* [View Parameters](#view-parameters)
+* [Example](#example)
+
+## View Parameters
+
+The following parameters allow you to configure the agenda view:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Attribute | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `NumberOfDays` | `int` <br /> (`7 (a week)`) | Represents the number of days shown in the view. |
+| `HideEmptyAgendaDays` | `bool` <br /> (`true`) | Defines whether dates with no appointments are rendered. |
+
+## Example
+
+>caption Declare the Agenda View in the markup
+
+>tip You can declare other views as well, this example adds only the Agenda view for brevity.
```

Comment:
Having a tip directly below the caption is not ideal. Consider moving the tip above the caption.